canonical-web-and-design/jaas.ai
# Speed up run

## Done

Speed up ./run

## QA

- Pull code
- Run `./run build` and then `./run build` again. At least the second time, check you see:

```
Yarn dependencies haven't changed.
```

Now try the new yarn command shortcut:

```
./run yarn build
./run yarn -p 8029 serve
```

Check they work as expected.
